test/test_runtime.py

```diff
@@ -87,6 +87,25 @@ def test_runtime_version_fail_cli(mocker: MockerFixture) -> None:
         runtime.version  # pylint: disable=pointless-statement
 
 
+def test_runtime_install_role(caplog) -> None:
+    """Checks that we can install roles."""
+    caplog.set_level(logging.INFO)
+    project_dir = os.path.join(
+        os.path.dirname(__file__), "assets", "ansible-role-mysql"
+    )
+    runtime = Runtime(isolated=True, project_dir=project_dir)
+    runtime.prepare_environment()
+    for record in caplog.records:
+        assert record.levelname != "CRITICAL"
+    assert "symlink to current repository in order to enable Ansible to find the role" in caplog.text
+    print(caplog.text)
+    # check that role appears as installed now
+    result = runtime.exec(["ansible-galaxy", "list", "-vvv"])
+    # print(result)
+    assert result.returncode == 0, result
```

Comment:
No need to print out the `result` like this — just add `--showlocals` to addopts to reveal it.